doc/beamerug-nonpresentation.tex

```diff
@@ -235,9 +235,9 @@ \subsubsection{Including Slides from the Presentation Version in the Article Ver
   \begin{quote}
     \meta{filename}|.page|\meta{page of label name}|.|\meta{extension}
   \end{quote}
-  For each page of your |.pdf| or |.ps| file that is to be included in this way, you must create such a file by hand. For example, if the PostScript file of your presentation version is named |main.beamer.ps| and you wish to include the slides with page numbers 2 and~3, you must create (single page) files |main.beamer.page2.ps| and |main.beamer.page3.ps| ``by hand'' (or using some script). If these files cannot be found, |pgf| will complain.
+  For each page of your |.pdf| or |.ps| file that is to be included in this way, you must create such a file by hand. For example, if the PostScript file of your presentation version is named |main_beamer.ps| and you wish to include the slides with page numbers 2 and~3, you must create (single page) files |main_beamer.page2.ps| and |main_beamer.page3.ps| ``by hand'' (or using some script). If these files cannot be found, |pgf| will complain.
```

Comment:
`main_beamer.page2.ps` is a bit inconsistent when trying to avoid multiple periods in names